comonad.cabal

```diff
@@ -68,6 +68,17 @@ flag distributive
   default: True
   manual: True
 
+flag indexed-traversable
```

Comment:
Does this flag make sense considering that `indexed-traversable` has only GHC deps?

Reply:
ditto `distributive` and `tagged` on recent enough GHCs. IMHO yes, for consistency. The implementation of this flag is quite cheap.